express-api/src/services/ches/chesServices.ts

```diff
@@ -136,19 +139,14 @@ const sendEmailAsync = async (email: IEmail, user: SSOUser): Promise<IEmailSentR
   email.from = email.from ?? cfg.ches.defaultFrom;
 
   if (cfg.ches.bccCurrentUser) {
-    email.bcc = [user.email, ...(email.bcc ?? [])];
+    email.bcc = [user?.email, ...(email.bcc ?? [])];
```

Comment:
By making user an optional argument to this function and accessing this field with the null conditional operator, we could run into cases where this introduces 'undefined' to the BCC list. Getting rid of this option altogether could be another option though, dunno if this actually gets used in live.

Reply:
Good point, I can remove it.

Reply:
I've reverted some changes, and the access request notification seems to be sending successfully. Might need someone else to test as well.